socorro/external/pubsub/crashqueue.py

```diff
@@ -118,15 +123,14 @@ def __iter__(self):
             self.priority_path,
             self.standard_path,
             self.reprocessing_path,
-            self.priority_path
```

Comment:
Since we switched to a system where we process everything we save, the likelihood there's a crash report that's unprocessed when someone views it on Crash Stats drops significantly. Given that, I took out checking the priority queue twice in a pass. That'll reduce pull requests that yield 0 messages which are effectively wasted work.